examples/iota_lens/input_iotalens.in

```diff
@@ -21,14 +21,9 @@ beam.mutpt = 0.0
 ###############################################################################
 # Beamline: lattice elements and segments
 ###############################################################################
-lattice.elements = beam_monitor
-                   const_end nllens const nllens const nllens const nllens const
+lattice.elements = const_end nllens const nllens const nllens const nllens const
                    nllens const nllens const nllens const nllens const nllens
                    const nllens const_end
-                   beam_monitor
-
-beam_monitor.type = BeamMonitor
-beam_monitor.backend = "h5"
 
```

Comment:
Yes, agreed. I currently leave it in but we should generalize it.

We can calculate it from the particle data, but in situ (but with custom options, so it only runs when we request it) is probably even better because it produces less data and is thus faster.